### PR TITLE
Adding Greeting component

### PR DESCRIPTION
**Pull request by amirpourpanahi**
### Description

List of proposed changes:

- Add greeting component

### CODE REVIEW

**Review by nabil386 — APPROVED** [review]

File: components/molecules/DSHeader.js [hunk 1]
```diff
@@ -18,6 +18,23 @@ export default function DSHeader() {
...
+            link: '/',
+            text: 'Canada.ca',
+          },
+          {
+            link: '/',
+            text: 'Service Canada',
+          },
+          {
+            link: '/',
+            text: 'My Benefits and Services',
```
Comment (s-laugh) [inline components/molecules/DSHeader.js:34]: The text and links will need to get translated, can we grab the values from the content files even if it's not translated yet? And also add the links for Canada.ca and Service Canada.

**Review by s-laugh — CHANGES_REQUESTED** [review]
Looks good, just a small tweak please :)

File: components/molecules/DSHeader.js (hunk 1 above)
Comment (amirpourpanahi) [reply components/molecules/DSHeader.js]: Done!

**Review by s-laugh — APPROVED** [review]
Awesome!

**Pull request merged**